src/App.scss

```diff
@@ -1 +1,59 @@
-// styles go here
+.Page {
```

Comment:
if you already decided to style it beautifuly, you should to bring it to the end) you main container is twitching when click on show\hide